src/wallet/rpc/wallet.cpp

```diff
@@ -846,6 +846,46 @@ static RPCHelpMan createwalletdescriptor()
     };
 }
 
+static RPCHelpMan exportwatchonlywallet()
+{
+    return RPCHelpMan{"exportwatchonlywallet",
+        "Creates a wallet file at the specified path and name containing a watchonly version "
+        "of the wallet. This watchonly wallet contains the wallet's public descriptors, "
+        "its transactions, and address book data. The watchonly wallet can be imported to "
+        "another node using 'restorewallet'.",
+        {
+            {"destination", RPCArg::Type::STR, RPCArg::Optional::NO, "The path to the filename the exported watchonly wallet will be saved to"},
+        },
+        RPCResult{
+            RPCResult::Type::OBJ, "", "",
+            {
+                {RPCResult::Type::STR, "exported_file", "The full path that the file has been exported to"},
+            },
+        },
+        RPCExamples{
+            HelpExampleCli("exportwatchonlywallet", "\"home\\user\\\"")
+            + HelpExampleRpc("exportwatchonlywallet", "\"home\\user\\\"")
```

Comment:
In commit "wallet, rpc: Add exportwatchonlywallet RPC" (71e25ee9ad52d8b9a763782b795690a196f218ef)

Seems like a mistake here, did you mean to write `"\"home\\user\\backup.dat\""`?